README.md

```diff
@@ -74,6 +74,67 @@ cd Much-Assembly-Required\target
 java -jar server-1.4a.jar
 ```
 
+## macOS (tested on Sierra 10.12.6)
+
+###Installation
+
+1. Install [Maven3](https://maven.apache.org/)
+   -Add Maven bin to your path
+   ```bash
+   export PATH=/path/to/maven/bin.:$PATH
+   ```
+2. Install [MongoDB](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/?_ga=2.201359831.774868398.1539369140-197602459.1539369140).
+   -Via Brew:
+   ```bash
+   #Update brew
+   brew update
+
+   #Install mongodb
+   brew install mongodb
+
+   #Install latest development release
+   brew install mongodb --devel
+   ```
+   -Via .tar.gz
+   ```bash
+   #Extract files:
+   tar -zxvf mongodb-osx-ssl-x86_64-4.0.3.tgz
+   
+   #Ensure binaries are in your path
+   export PATH=<mongodb-install-directory>/bin:$PATH
+   ```
+If you do not wish to use the default data directory (/data/db), follow the steps for running MongoDB in the install doc.
+
+### Deploying Server
```

Comment:
Could you change the 'installation' and 'deploying the server' section headers from `### Deploying Server` to `**Deploying Server**` please?

Reply:
Sure thing! One moment...